includes/Services/PluginService.php

```diff
@@ -21,6 +22,28 @@
  * Class for providing plugin related services.
  */
 class PluginService {
+
+	/**
+	 * Gets the list of plugins based on different enabled Hiive Flags
+	 *
+	 * @return array
+	 */
+	public static function initialize_hiive_flag_plugins() {
+		$init_plugins_extended = array();
+		$hiive_flags           = Plugins::get_hiive_plugin_flags();
```

Comment:
I think we can skip the extra `enablement` flag for the Hiive flags. I don't see a benefit in disabling them if the capability is already set. Essentially, we just move the plugins to a capability-to-plugins map and use it directly if the capability is set.